libexec/python/utils.py

```diff
@@ -57,13 +56,13 @@ def add_http(url,use_https=True):
     prefix = "https://"
     if use_https == False:
         prefix="http://"
-    
+
     # Does the url have http?
-    if re.search('^http*',url) == None:
+    if not url.startswith('http'):
```

Comment:
I think we would want the developer using the function to be able to be specific about this, which is why I first had the `use_https` boolean. The bug here (that I mentioned earlier) is that if a url goes in that is not https, and the user specifies wanting https, then the call above using `startswith` is going to be True, and move on to return the original `http` url. My suggestion before was to do the check specifically for the prefix that was asked for by the user, and regardless of what the url looks like, the function should absolutely return what was asked for.